unlock-protocol-com/blog/grants-program-updates.md

```diff
@@ -0,0 +1,77 @@
+[applying](https://docs.unlock-protocol.com/governance/grants-bounties/udt-grantee-handbook#submitting-an-application). 
+
+In addition, our new processes should make adding new [grant committee](https://docs.unlock-protocol.com/governance/grants-bounties/grant-committee-applicant-guide) 
+members much easier. This will enable Unlock to scale the program as the ecosystem grows. 
+
+Using the experience gained from the over 100 grant applications we received to date, we sought to increase the impact of the projects we give grants to and have made improvements to the 
+[grant review](https://docs.unlock-protocol.com/governance/grants-bounties/grant-review-guide) guidance for the committee. 
+
+### What’s next?
+
+We’re thrilled so far with all of these improvements and we’re confident this next season of grants will be even more effective at increasing network volume and adding to the credibility and visibility of Unlock Protocol. On a quarterly basis, the grants committee will review the progress of the program and make proposals to the community for iterative improvements, if needed. This will accelerate the success of the program while maintaining our commitment to decentralization. 
+
+👉 Interested in building with us? Fill out an [application](https://docs.unlock-protocol.com/governance/grants-bounties/udt-grantee-handbook#submitting-an-application) for one of grants now! 
+
+### Looking back at the last year: Unlock Protocol grant success stories
+
+Over the past year, the grants program has had success spawning work in areas of plugins and integrations, education, showcases, events, and platform subscriptions. Some of the highlights include:
+
+**Integrations & Plugins**
+
+- [Guild.xyz](https://unlock-protocol.com/guides/guild-xyz/)
+- [Unity Game Engine](https://github.com/thehen/unlock-unity-package)
+- [Monetized](https://unlock-protocol.com/guides/how-to-token-gate-notion/) (Notion)
+- [Collab.Land](https://unlock-protocol.com/guides/discord-with-collab-land/)
+- [Decentraland](https://unlock-protocol.com/blog/decentraland)
+- [Firebase](https://unlock-protocol.com/blog/firebase-integration)
+- [Discourse](https://unlock-protocol.com/blog/discourse-plugin)
+- [Drupal](https://www.drupal.org/project/unlock)
+
+**Built with Unlock**
+
+- [Tales of Elatora](https://unlock-protocol.com/blog/talesofelatora)
+- [Content Rooms](https://contentrooms.com/)
+- [Bonsai.so](https://bonsai.so/)
+- [Dinner DAO](https://unlock-protocol.com/blog/dinnerdao)
+- [Unfurl](https://unlock-protocol.com/blog/unfurl-connect)
+
+**Events**
+
+- [ETHAnglia](https://ethanglia.org/)
+
+**Education**
+
+- [Web3Academy](https://w3academy.io/)
+- [dProgramming University](https://dprogramminguniversity.com/courses/unlock-protocol-101/)
+- [Unlock Showcase](https://www.unlockshowcase.com/)
+
+With the upgraded grants program, we can’t wait to see what the community creates next! You can fill out a grant application here!
```

Comment:
`fill out a grant application here` should link to https://docs.unlock-protocol.com/governance/grants-bounties/udt-grantee-handbook/#submitting-an-application